server/src/test/java/com/defold/extender/AuthenticationTest.java

```diff
@@ -31,7 +31,7 @@
 public class AuthenticationTest {
 
     private static final int EXTENDER_PORT = 9000;
-    private static final String SDK_VERSION = "d31d6397a72178541a5ef6e7ef2bed090d828f58"; // 1.2.190
+    private static final String SDK_VERSION = "d393bae6a361f86cf2263ab312c9b3cea45253ab"; // 1.2.191
```

Comment:
I get "can't find SDK" with 1.2.190. 
So, I updated version and now tests are fine

Reply:
It's a bit strange though, as the sdk should be available?
http://d.defold.com/stable/

Reply:
I agree, it is strange.
but I decided that the simplest way to fix that is to use new version (there are no big difference in this sense between 190 and 191)